python_backend/agents/base_agent.py

```
@@ -0,0 +1,327 @@
+            result = await self._process_final_result(final_state)
+            result.processing_time = time.time() - start_time
+            
+            self.logger.info(f"{self.config.name} 실행 완료 - 소요시간: {result.processing_time:.2f}초")
+            return result
+            
+        except asyncio.TimeoutError:
+            error_msg = f"{self.config.name} 실행 타임아웃 ({self.config.timeout}초)"
+            self.logger.error(error_msg)
+            return self._create_error_result(error_msg, time.time() - start_time)
+        
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)